spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+RSpec.describe VehicleFactory do
+  describe '#create_vehicles' do
+    it 'create an array of vehicles from the given data' do
+      factory = VehicleFactory.new
+      data = [{:make=>"Tesla", :model=>"Model Y", :vin_1_10=>"7SAYGDEE8P", :model_year=>"2023"},
```

Comment:
This is not quite the data we'd be feeding into these methods.  Again, this would be coming from the api data.